### PR TITLE
BMD-6296 add user cleanup list

### PR DESCRIPTION
**Pull request by bmxpiku**
### Jira ticket:
https://berlingskemedia.atlassian.net/browse/BMD-6296

### CODE REVIEW

File: client/app/components/mdbapi.service.js [hunk 1]
```diff
@@ -143,6 +143,17 @@
...
 
+          service.getInactiveUsers = function (params) {
+            return $http.get(baseurl + "/users/not-active-list")
+              .then(_httpSuccessCallback);
+          };
+
+          service.deleteInactiveUsers = function (params) {
+            params.location_id = params.location_id !== undefined ? params.location_id : LOCATIONID;
+            return $http.delete(baseurl + "/users/not-active-list")
+              .then(_httpSuccessCallback);
```
Comment (dgolabek-sii) [inline client/app/components/mdbapi.service.js:154]: you are not using those params, are you?

File: client/app/cleanup/user.controller.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('nyhedsbreveprofiladmin')
+    .controller('InactiveUsersController', InactiveUsersController);
+
+  /** @ngInject */
+  function InactiveUsersController($scope, $state, mdbApiService, toastr, authResolved) {
+    var vm = this;
```
Comment (dgolabek-sii) [inline client/app/cleanup/user.controller.js:10]: var?

File: client/app/cleanup/user.controller.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        console.log('Success');
+        location.reload();
+      });
+    }
+
+    function activate() {
+      vm.didSearch = false;
+      $scope.search = search;
+      $scope.delete = deleteUsers;
+      console.log('eloszka ziomeczki');
```
Comment (dgolabek-sii) [inline client/app/cleanup/user.controller.js:41]: console logs? :)

File: client/app/cleanup/user.controller.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      vm.searching = true;
+      mdbApiService.getInactiveUsers().then(function(users) {
+        vm.users = users;
+        vm.didSearch = true;
+        vm.searching = false;
+      });
+    }
+
+    function deleteUsers() {
+      console.log('I hope you know what are you doing');
```
Comment (dgolabek-sii) [inline client/app/cleanup/user.controller.js:30]: user doesn't see that "alert" :P

**Review by dgolabek-sii — CHANGES_REQUESTED** [review]

File: client/app/cleanup/user.controller.js (hunk 4 above)
Comment (bmxpiku) [reply client/app/cleanup/user.controller.js:30]: i know :D

File: client/app/cleanup/user.list.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+<div class="clearfix">
+  <div class="alert alert-warning" role="alert" ng-if="vm.didSearch && vm.users.length === 0">
+    <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+    No results found
+  </div>
+</div>
+
+
+<!-- Modal -->
+<div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```
Comment (dgolabek-sii) [inline client/app/cleanup/user.list.html:43]: exampleModal ;-)

**Review by dgolabek-sii — CHANGES_REQUESTED** [review]

**Review by dgolabek-sii — APPROVED** [review]

**Review by bmxmaza — APPROVED** [review]

**Pull request merged**